kolibri/plugins/learn/assets/src/views/CategorySearchModal/CategorySearchOptions.vue

```diff
@@ -6,7 +6,7 @@
         :text="coreString(camelCase(selectedCategory))"
         :appearanceOverrides="appearanceOverrides"
         appearance="basic-link"
-
+        :disabled="availablePaths && !availablePaths[nestedObject.value]"
```

Comment:
I think this should be:
```suggestion
        :disabled="availablePaths && !availablePaths[topLevelCategory.value]"
```